guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -56,24 +56,22 @@ For external database servers: `/var/lib/pgsql` with installation size of 100 MB
 |/opt/puppetlabs | 500 MB | Not Applicable
 
 ifdef::katello,satellite,orcharhino[]
-|/var/lib/pulp/ |1 MB |300 GB
+|/var/lib/pulp |1 MB |300 GB
 
-|/var/lib/qpidd/ |25 MB | Not Applicable
+|/var/lib/qpidd |25 MB | xref:storage-guidelines_{context}[Refer Storage Guidelines]
 endif::[]
 |====
 endif::[]
 
-For external database servers: `/var/lib/pgsql` with installation size of 100 MB and runtime size of 20 GB.
-
```

Comment:
This reverts the changes made in https://github.com/theforeman/foreman-documentation/pull/1684, which seems wrong.

Reply:
Oww. Yes. Thank you for bringing out this. When I raised this PR, the mentioned PR #1684 wasn't merged I guess. It would have been rebased. 

Addressed :+1: